html/home-page.html

```diff
@@ -0,0 +1,167 @@
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </div>
+    </header>
+    <main>
+        
+        <section class="container-fluid head">
+            <div class="container headline">
+            <h1 class="greetings">Hello! World</h1>
+        <h1 class="title"> Let's built  from <br>here</h1>
+        <p class="narates"> GitHub Universe 2022 is in the books but you’re still able to view sessions from all our content tracks. If you registered for the event you can login below, or sign-up as a first time viewer for our session catalog.</p>
+        <h2 class="date">2022.11.09(WED)~10(THU)</h2>
+        <h3 class="venu">Virtual Schedule @watch.githubuniverse.com</h3>
+            </div>
+            <img src="../images/Hero_Gradient.webp" alt="">
+        </section>
+
+        <article class="container-fluid pg ">
+        <div class="main">
+        <h3 class="main-title">Main Program</h3>
+        
+        <span class="line"></span>
+        
+        <div class="container Program-content">
+                <ul>
+                    <li>
+                        <img src="../images/asterisk_0.svg" alt="cloud pic">
+                        <a href="#">Software</a>
+                        <p>Learn how the best companies, developers, and start-ups are using software in the cloud to accelerate productivity form their business. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_1.svg" alt="security pic">
+                        <a href="#">Security</a>
+                        <p>Hone your security skills by exploring the GitHub tools, techniques, and best practices for securing your software development lifecycle. 
+                        </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                </ul>
+                
+         </div>
```

Comment:
- Just like the previous reviewer pointed out, please ensure that your logo and text are aligned as they are in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

<img width="174" alt="wass" src="https://user-images.githubusercontent.com/73607512/203768502-c93b4d69-56d0-403a-9f97-da3a0449dced.PNG">

**OPTIONAL** - You have 3 cards that look alike, I suggest that all your cards are not the same and are unidentical. Please fix this 🙏 🙏

Reply:
![Screenshot 2022-11-24 164157](https://user-images.githubusercontent.com/110728909/203776211-5715ca7a-c198-4039-9243-209522b37d03.png)

i am sorry according to my code everything is fine. i have attach my screen shot as well .I think you have open the wrong code since screen shot you added is not correct. here it says
<li>
                        <img src="../images/asterisk_2.svg" alt="AI pic">
                        `<a href="#">Artificial</a>`
                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
                    </li>
but screenshot attached show the wrong content means that you have opened wrong code.